[Bug]: Looking for easypost.beta

### Software Version

7.1.0

### Language Version

Python 3.10

### Operating System

Docker (Linux)

### What happened?

1. Running a new instance of CI/CD with tests that had previously passed.
2. Received the below error (appears to be looking for `easypost.beta`)


### What was expected?

No error.

### Sample Code

```python3
`import easypost`
```


### Relevant logs

```sh
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
shipping/tests/test_easypost.py:14: in <module>
    from shipping import easypost_client as ep
shipping/easypost_client.py:10: in <module>
    import easypost
/opt/hostedtoolcache/Python/3.10.4/x64/lib/python3.10/site-packages/easypost/__init__.py:1: in <module>
    import easypost.beta
E   ModuleNotFoundError: No module named 'easypost.beta'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v7.1.1 (2022-05-09)
+
+- Fixes the inclusion of the new `beta` module
+
 ## v7.1.0 (2022-05-09)
 
 - Adds a `lowest_rate()` function to Orders and Pickups

--- a/easypost/version.py
+++ b/easypost/version.py
@@ -1,4 +1,4 @@
-VERSION = "7.1.0"
+VERSION = "7.1.1"
 
 elements = VERSION.split("-")
 numbers = [str(v) for v in elements[0].split(".")]

--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,7 @@
-from setuptools import setup
+from setuptools import (
+    find_packages,
+    setup,
+)
 
 
 REQUIREMENTS = [
@@ -30,12 +33,17 @@ with open("README.md", encoding="utf-8") as f:
 
 setup(
     name="easypost",
-    version="7.1.0",
+    version="7.1.1",
     description="EasyPost Shipping API Client Library for Python",
     author="EasyPost",
     author_email="support@easypost.com",
     url="https://easypost.com/",
-    packages=["easypost"],
+    packages=find_packages(
+        exclude=[
+            "examples",
+            "tests",
+        ]
+    ),
     install_requires=REQUIREMENTS,
     extras_require={
         "dev": DEV_REQUIREMENTS + CPYTHON_DEV_REQUIREMENTS,

